src/MicrosoftTeams.ts

```diff
@@ -417,26 +428,52 @@ namespace microsoftTeams
         {
             ensureInitialized(frameContexts.content, frameContexts.settings, frameContexts.remove);
 
+            authParams = authenticateParameters;
+
```

Comment:
I do close the existing window if there is one; however, I am not certain that we want to call notifyFailure because it might confuse the hell out of developers who reuse the same error handler across authenticate calls.